bigquery/indexer.py

```diff
@@ -267,8 +267,33 @@ def index_fields(es, index_name, table, sample_id_column):
     for field in fields:
         if field.name == sample_id_column:
             id_prefix = "samples." + id_prefix
+    mappings = {
```

Comment:
// Use simple analyzer so underscores are treated as a word delimiter.
// With default analyzer, searching for "baseline" would not find BQ column named "age_at_baseline".
// With simple analyzer searching for "baseline" would find BQ column named "age_at_baseline".